sdk/ai/azure-ai-projects/tests/agents/test_agents_client.py

```diff
@@ -45,13 +46,13 @@
 
 agentClientPreparer = functools.partial(
     EnvironmentVariableLoader,
-    "azure_ai_project",
-    project_connection_string_agents_tests="https://foo.bar.some-domain.ms;00000000-0000-0000-0000-000000000000;rg-resour-cegr-oupfoo1;abcd-abcdabcdabcda-abcdefghijklm",
+    "azure_ai_projects",
+    azure_ai_projects_connection_string_agents_tests="foo.bar.some-domain.ms;00000000-0000-0000-0000-000000000000;rg-resour-cegr-oupfoo1;abcd-abcdabcdabcda-abcdefghijklm",
```

Comment:
Yes, per your comment, please update this to `azure_ai_projects_agents_tests_project_connection_string` (corresponding to env variable `AZURE_AI_PROJECTS_AGENTS_TESTS_PROJECT_CONNECTION_STRING`)